source/routes/create/daily.js

```diff
@@ -0,0 +1,47 @@
+const PouchDB = require('pouchdb');
```

Comment:
This is from daily CRUD - also should be removed, as it will be committed in the daily pull request.